README.md

```diff
@@ -164,6 +173,30 @@ Then(I: see(theScreen: .settings))
 - `scroll`: Scrolls to a given element, the default is to scroll down but this is overridable using a `bool` argument flag.
 - `visible`: Whether the element is visible.
 
+### Biometrics
+
+`TABTestKit` allows you to simulate biometrics on the simulator. This means you can enroll or unenroll biometrics and simulate a successful or unsuccessful biometric authentication. This can be really helpful when trying to automate biometric features of your app.
```

Comment:
Worth mentioning about the Face ID permissions prompt? Especially since I didn't catch it!